tests/dynamic_sim_type_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+u"""Using a sim type outside of Sirepo
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import contextlib
+import pytest
+
+
+def test_run():
+    from pykern import pkunit
+    from pykern.pkdebug import pkdp, pkdlog
+
+    with _rsprivate():
+        fc = _fc()
+        r = fc.sr_login_as_guest()
+        d = fc.sr_sim_data(sim_type='code1', sim_name='Secret sauce')
+        pkunit.pkeq('green', d.models.sauce.color)
+
+
+def _fc():
+    from pykern.pkdebug import pkdp
+    from pykern.pkcollections import PKDict
+    from sirepo import srunit
+
+    PKDict(
+        SIREPO_FEATURE_CONFIG_DYNAMIC_SIM_TYPES='code1:rsprivate',
+    )
+    fc = srunit.flask_client(
+        cfg=PKDict(
+            SIREPO_FEATURE_CONFIG_DYNAMIC_SIM_TYPES='code1:rsprivate',
+        ),
+        no_chdir_work=True,
+    )
+    return fc
+
+
+@contextlib.contextmanager
+def _rsprivate():
```

Comment:
@robnagler thoughts on this? I don't know if there is a more elegant way for installing a package just for a test.

Reply:
I think you might want to use PYTHONPATH so you don't have to install globally, which is not the best. At a minimum, the name of the package should be something like sirepo_test_dynamics_sim_type or something like that so we know it is used for testing and for which package.

Reply:
6e28eb52